rotkehlchen/history/events/structures/asset_movement.py

```diff
@@ -228,6 +228,7 @@ def create_asset_movement_with_fee(
         fee_asset: Asset,
         fee: 'FVal',
         unique_id: str | None = None,
+        identifier: int | None = None,
```

Comment:
Where do you need this?

Reply:
this is used when creating the `AssetMovement` instances inside the api schema